guides/doc-Updating_Project/master.adoc

```diff
@@ -0,0 +1,12 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:updating:
+:context: updating
+
+ifdef::satellite[]
+= {UpdatingDocTitle}
+
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+
+include::common/assembly_updating-server-smart-proxy.adoc[leveloffset=+1]
```

Comment:
Does this not apply to Foreman/Katello/orcharhino? I lean towards adding this for all build targets unless there's specific reasons not to do this. Speaking for orcharhino, I'd happily follow you/RH Satellite here.

Reply:
I didn't found updating guide for any other projects except Satellite. So, kept as it is. But, I can add Orcharhino. Need confirmation for other projects.